client/index.ejs

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html ng-app="app" class="layout-pf layout-pf-fixed layout-pf-fixed-inner-scroll transitions">
+<head lang="en">
+  <title ng-bind="title" translate>Self Service</title>
+  <meta charset="utf-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1"/>
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"/>
+  <link rel="shortcut icon" href="/favicon.ico"/>
+  <link rel="icon" href="/favicon.ico"/>
+  <base href="/self_service/">
+  <style>
+    [ng-cloak] {
+      display: none;
+    }
+
+    [ng-cloak].splash {
+      display: block !important;
+    }
+
+    .splash {
+      display: none;
+      background-color: #428bca;
+    }
+  </style>
+
+  <!-- build:css styles/lib.css -->
```

Comment:
So these are still used by build process, we run wiredep over index.html, literally exactly the same as before.

there is a cleaner way to do it, kill that stuff use [gulp-ejs](https://www.npmjs.com/package/gulp-ejs) to inject those dependency partials, but i'll be honest... didn't want to put the time into figuring out which older version would jive with our node version and not cause issue.

if ya want @himdel i can make an outstanding issue to do that conversion whenever we update everything (the conversion of removing the build comments and relying wholly on ejs for dependency injection during the build process)